lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -6,6 +6,7 @@ SQLAlchemy==1.0.15
 mercurial==3.7.3
 numpy==1.9.2
 pycrypto==2.6.1
+pyuwsgi==2.1.dev0+gx1.c6ee1f6
```

Comment:
https://github.com/galaxyproject/starforge/pull/114/files#diff-2fbe7e1ff37f22c3de4b47d45e196667R101 is using the uWSGI name, should that be changed then?